### ISSUE TITLE
Confusing comment regarding apple touch icon

### ISSUE THREAD

**Issue by nathanieloffer** [h5bp/html5-boilerplate#1783]
The image in the <link> statement is named apple-touch-icon.png but the comment refers to favicon.ico. This should be changed to improve readability especially for new web developers.

`<link rel="apple-touch-icon" href="apple-touch-icon.png">`
     `<!-- Place favicon.ico in the root directory -->`

**Comment by rsp** [comment]
@nathanieloffer My understanding of that is that the apple touch icon is named explicitly and thus it doesn't need a comment on where to put it (like normalize.css or other resources) and that comment refers to a favicon because it isn't explicitly mentioned anywhere else than in that comment. I may be wrong but this is how I understand it.

**Comment by chrislaughlin** [comment]
If the touch icon is the favcon then maybe comment should also mention this.

**Comment by chrislaughlin** [comment]
Addition to last comment, the touch icon is used for web clipping on iOS, I guess a comment would be useful here for new developers

**Comment by roblarsen** [comment]
These are two separate elements. They're related by being icons and by being next to each other, but they're two separate elements.  Specifically, the comment isn't about the apple-touch-icon. The comment is _an instruction_ about how to implement a favicon. favicon.ico is placed in the root directory because it's automatically requested by browsers. You don't need one in the source. 

The comment (in hindsight) is poor because presupposes that a developer expects to see `<link rel="shortcut icon" href="/favicon.ico">` in the head of the document.

**Comment by nathanieloffer** [comment]
You're right roblarsen I was expecting to see that <link rel> line from days gone by. Hence my confusion regarding the comment.
